astroquery/gaia/core.py

```diff
@@ -757,9 +750,9 @@ def load_user(self, user_id, *, verbose=False):
         return self.is_valid_user(user_id=user_id,
                                   verbose=verbose)
 
-    def cross_match(self, *, full_qualified_table_name_a=None,
-                    full_qualified_table_name_b=None,
-                    results_table_name=None,
+    def cross_match(self, *, full_qualified_table_name_a,
```

Comment:
yes, I suppose we could have kept these positional args, but I think there is no need to change them now.